packages/components/src/Banner/Banner.mdx

```diff
@@ -30,7 +30,8 @@ import { Banner } from "@jobber/components/Banner";
 
 <Props of={Banner} />
 
----
+Banner type should be one of the following "notice" | "success" | "warning" |
+"error".
```

Comment:
For this, we have the different `types` below in the `mdx`. We may want to change this to be a more readable header with sub-headings for each type.

Thoughts @chris-at-jobber

Reply:
How about this 
![image](https://user-images.githubusercontent.com/84454304/134960083-7459a1c6-eab0-4ad7-bac2-df40733676f9.png)

Reply:
👍  basically just pushed this up but with links to each type's usage guidelines

Reply:
Also nests the usage guidelines for the respective types under this new header!

Reply:
@chris-at-jobber  Done! I've added links too.